Document users are responsible to keep external discovery CM up-to-date (#400)
stackabletech/documentation: modules/concepts/pages/service_discovery.adoc

  ...
----

> IMPORTANT: It's important that you provide *all* the information in the discovery CM and that they are up-to-date!
> For Stackable managed services the Stackable operators will take care of this, when discovering external services it's your responsibility.
> 

Reviewer: ```suggestion
IMPORTANT: It's important that you provide *all* the information in the discovery ConfigMap and that it is up-to-date!
For Stackable managed services the Stackable Operators will take care of this, but for external service discovery it's your responsibility.

```
